x-pack/plugins/alerting/server/routes/rule/apis/bulk_enable/bulk_enable_rules_route.test.ts

```diff
@@ -208,7 +210,9 @@ describe('bulkEnableRulesRoute', () => {
       bulkEnableRulesRoute({ router, licenseState });
       const [_, handler] = router.patch.mock.calls[0];
 
-      rulesClient.bulkEnableRules.mockResolvedValueOnce(bulkEnableActionsResult);
+      rulesClient.bulkEnableRules.mockResolvedValueOnce(
+        bulkEnableActionsResult as unknown as BulkEnableRulesResult
```

Comment:
Having to do this because we were so lazy with mocking rules in our tests that our types were so bad that it allowed it to work. 

I could fix it but i feel like thats out of the scope of this PR

Reply:
I believe it is easier to do small fixes while we version rules. Otherwise, they will pile up to a point where it will be hard to do it and probably we will forget to do it. Fixing tests feels like part of this PR because we change the types and tests should follow. I am advocating fixing everything just things that will help us prevent bugs in the future. If the fix involves fixing code unrelated to the `rulesClient.bulkEnableRules` like the issue you said about `SanitizedRule ` then at least we can type the `bulkEnableActionsResult` so in the future will warn us that we need to take into account more attributes. Wdyt?